bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/model/jsonadapter/MetricsEventAdapterFactory.kt

```diff
@@ -96,6 +97,11 @@ class MetricsEventAdapterFactory : JsonAdapter.Factory {
           }
         }
 
+        if (value.sdk_version != null) {
```

Comment:
sdk_version field does not exist in the request when the event is tracked with older SDK version